packages/components/src/Typeahead/Typeahead.scss

```diff
@@ -95,6 +95,7 @@ $tc-typeahead-items-font-size: 1.5rem !default;
 			position: absolute;
 			margin: 0;
 			right: 0;
+			bottom: 0.6rem;
```

Comment:
it seems to be a magic number, could you explain why 6px is needed here, please?

Reply:
no any magic here

it's just a bottom position, which needs to be specified for IE11
just "right" position seems not enough for IE11 and it sets the bottom to 0 by default, that's why it's positioned at the bottom of container with position relative, and not centered

0.6rem is the value that centers it vetically in Typeahead container, decided to specify it in rems since the width and height for icon are also in rems

(tested in all browsers)

Reply:
By magic number, he means that 0.6rem is not normalized and is really specific to a context. So if we use this component in another context, where we want it to have another height for example, it doesn't work. For example if we want to put that in a headerbar taking all the height.
Center things vertically should nnot be done with magic number

Reply:
with the last changes the caret will be centered vertically, regardless of height

although it won't  be if we are going to change the caret size in some contexts, but if are going to - then the caret width and height will need to be overwritten anyway, and so margin can be adjusted